uint8 player_id
uint8 bet
---
bool result
